quarkchain/evm/state.py

```diff
@@ -317,8 +322,15 @@ def get_and_cache_account(self, address):
         o._cached_rlp = None
         return o
 
-    def get_balance(self, address):
-        return self.get_and_cache_account(utils.normalize_address(address)).balance
+    def get_balances(self, address):
+        return self.get_and_cache_account(
+            utils.normalize_address(address)
+        ).token_balances.balances
+
+    def get_token_balance(self, address, token_id):
+        return self.get_and_cache_account(
+            utils.normalize_address(address)
+        ).token_balances.balance(token_id)
```

Comment:
should be `balance[token_id]`?

Reply:
`balance()` is a function, either that or `balances[token_id]`, but the latter does not handle the case where the token in question has been added to the account before